apis/python/node/src/lib.rs

```diff
@@ -387,7 +387,12 @@ pub fn run(dataflow_path: String, uv: Option<bool>) -> eyre::Result<()> {
         .enable_all()
         .build()
         .context("tokio runtime failed")?;
-    let result = rt.block_on(Daemon::run_dataflow(&dataflow_path, uv.unwrap_or_default()))?;
+    let dora_command = DoraCommand::from_python_env()?;
```

Comment:
FYI, this is suppose to be run within a python script unrelated to the cli.

I'm slightly confused, why do we need dora command in this case. Is this to find the runtime python path?

Reply:
We need to be able to invoke the Dora CLI when there are any operators present in the dataflow. Operators are run by a `dora runtime` process, which we need to spawn.

This spawn command differs based on how Dora was installed. If it was installed as a normal binary,  we can use `current_exe` to get the path to the binary. For Python-based installations, this points to a `python` executable instead.

Either way, you're totally right that this is the wrong approach. Dora might still be installed as a normal binary even if the run command is invoked from Python. Or even worse, Dora might not be installed at all....